.circleci/config.yml

```diff
@@ -52,8 +52,20 @@ commands:
           curl -s --fail --location https://dl.google.com/go/go<<parameters.go_version>>.<<parameters.go_os>>-<<parameters.go_arch>>.tar.gz | tar -xz -C $HOME
           echo 'export PATH=$HOME/go/bin:$PATH' >> $BASH_ENV
           # if GOPATH is not set, `golang-ci` fails with an obscure message
-          # "ERRO Running error: context loading failed: failed to load program with go/packages: could not determine GOARCH and Go compiler"
+          # "ERROR Running error: context loading failed: failed to load program with go/packages: could not determine GOARCH and Go compiler"
           echo 'export GOPATH=$HOME/go' >> $BASH_ENV
+    - run:
+        # Without specifying default dns resolvers, iptables-nft interfere with
+        # our transparent proxy on the last dns hop, which makes the whole DNS
+        # query to fail
```

Comment:
Looking at this comment is this something we should add to the docs?

Reply:
And I was wrong. This change was unnecessary, so I removed it.